pkg/odo/cli/url/list.go

```diff
@@ -55,8 +56,9 @@ func NewURLListOptions() *URLListOptions {
 
 // Complete completes URLListOptions after they've been Listed
 func (o *URLListOptions) Complete(name string, cmd *cobra.Command, args []string) (err error) {
-	o.isDevfile = util.CheckPathExists(filepath.Join(o.componentContext, component.DevfilePath))
-	if o.isDevfile {
+	o.devfilePath = clicomponent.DevfilePath
```

Comment:
join the filepath with o.componentContext here

Reply:
updated